src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -424,6 +436,36 @@ private MetaInfo parseStatement(String sql) throws SQLServerException {
 
         return null;
     }
+    
+    private String removeCommentsInTheBeginning(String sql, int startCommentMarkCount, int endCommentMarkCount, String startMark, String endMark) {
+        int startCommentMarkIndex = sql.indexOf(startMark);
+        int endCommentMarkIndex = sql.indexOf(endMark);
+
```

Comment:
also, consider the case where there is no end comment. It will run into StringIndexOutOfBoundsException, drive has to handle it.